tools/clt_pickvalue.cwl

```diff
@@ -0,0 +1,21 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: clt_pickvalue
+doc: |
+  Given a file return that file. Used to simplify some pickvalue scenarios.
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: $(inputs.cpu)
+    ramMin: $(inputs.ram * 1000)
+baseCommand: [echo, done]
+inputs:
+  infile: { type: 'File' }
+  cpu: { type: 'int?', default: 8, doc: "CPUs to allocate to this task." }
+  ram: { type: 'int?', default: 16, doc: "RAM (in GB) to allocate to this task." }
```

Comment:
These seem like high compute requirements. Maybe drop them to 1 and 1?

Reply:
This is mostly to have it end up on a c5 instance by default. Mostly to limit IO time. Can be adjusted down in a workflow if necessary.